src/views/CountryRiskWatch/MultiMonthSelectInput/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { useCallback, useEffect, useRef } from 'react';
+import { _cs, listToMap } from '@togglecorp/fujs';
+import type { SetValueArg } from '@togglecorp/toggle-form';
+
+import RawButton from '#components/RawButton';
+
+import styles from './styles.module.css';
+
+const date = new Date();
+date.setDate(1);
+date.setMonth(0);
+
+const keyList = Array.from(Array(12).keys());
```

Comment:
Let's maybe use fill?

Reply:
No, `Array.fill` is only used when we need same value filled throughout the array